client/src/home/Home.js

```diff
@@ -19,10 +23,59 @@ const Home = () => {
     }
   }, [cookies.token]);
 
+  // Sort items by Alphabetical order
+  const sortAlpha = (a, b) => {
+    return a.name > b.name;
+  }
+
+  // Sort items by Expiry order
+  const sortExpiry = (a, b) => {
+    a = new Date(a.outDate);
+    b = new Date(b.outDate);
+    let diff = a.getTime() - b.getTime();
+    if (diff !== 0) {
+      return diff;
+    }
+    
+    // If time is the same, sort by alphabetical order
+    return a.name > b.name;
+  }
+
+  // Retrieve all items from MongoDB 
+  useEffect(() => {
+    getAllItems(cookies.token).then(items_raw => {
+      console.log(items_raw);
+      var batches_raw = [];
+      var itemNames = [];
+
+      if (items_raw != null) {
+        items_raw.forEach(item => {
+          // Format item names
+          item.name = item.name.charAt(0).toUpperCase() + item.name.slice(1);
+          itemNames.push({
+            id: itemNames.length,
```

Comment:
I'm just curious, what is this id for, and why did you set it to the current length of the itemNames array?

Reply:
The autocomplete package required a unique ID for each item